Transformation fail with can't create mutable bitmap with Config.HARDWARE in Picasso 3

Using: 
Picasso `com.squareup.picasso3:picasso:3.0.0-SNAPSHOT`
Pixel 2, Android 9

When trying to use a transformation, I am running into the following issue:
```
    java.lang.RuntimeException: Transformation rounded(radius=140.0, margin=0.0) crashed with exception.
        at com.squareup.picasso3.BitmapHunter$4.run(BitmapHunter.java:392)
        at android.os.Handler.handleCallback(Handler.java:873)
        at android.os.Handler.dispatchMessage(Handler.java:99)
        at android.os.Looper.loop(Looper.java:193)
        at android.app.ActivityThread.main(ActivityThread.java:6718)
        at java.lang.reflect.Method.invoke(Native Method)
        at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:493)
        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:858)
     Caused by: java.lang.IllegalArgumentException: can't create mutable bitmap with Config.HARDWARE
        at android.graphics.Bitmap.createBitmap(Bitmap.java:1036)
        at android.graphics.Bitmap.createBitmap(Bitmap.java:1000)
        at android.graphics.Bitmap.createBitmap(Bitmap.java:950)
        at android.graphics.Bitmap.createBitmap(Bitmap.java:911)
        at com.nicolasmilliard.playground.ui.util.RoundedTransformation.transform(RoundedTransformation.kt:11)
        at com.squareup.picasso3.BitmapHunter.applyTransformations(BitmapHunter.java:384)
        at com.squareup.picasso3.BitmapHunter.hunt(BitmapHunter.java:204)
        at com.squareup.picasso3.BitmapHunter.run(BitmapHunter.java:106)
        at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:458)
        at java.util.concurrent.FutureTask.run(FutureTask.java:266)
        at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1167)
        at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:641)
        at java.lang.Thread.run(Thread.java:764)
```
Using a config or the default result in the same issue.

Sample Project
https://github.com/niqo01/android-arch-playground/tree/nm/picassoBug

See Picasso usage
https://github.com/niqo01/android-arch-playground/blob/nm/picassoBug/app/src/main/java/com/nicolasmilliard/playground/ui/home/ItemResultViewHolder.kt#L54
and Tranformation:
https://github.com/niqo01/android-arch-playground/blob/nm/picassoBug/app/src/main/java/com/nicolasmilliard/playground/ui/util/RoundedTransformation.kt

--- a/picasso/src/main/java/com/squareup/picasso3/BitmapUtils.kt
+++ b/picasso/src/main/java/com/squareup/picasso3/BitmapUtils.kt
@@ -195,6 +195,7 @@ object BitmapUtils {
   @Throws(IOException::class)
   private fun decodeImageSource(imageSource: ImageDecoder.Source, request: Request): Bitmap {
     return ImageDecoder.decodeBitmap(imageSource) { imageDecoder, imageInfo, source ->
+      imageDecoder.isMutableRequired = true
       if (request.hasSize()) {
         val size = imageInfo.size
         if (shouldResize(

